diff --git a/.venv/Lib/site-packages/muxtools/muxing/tmdb.py b/.venv/Lib/site-packages/muxtools/muxing/tmdb.py
index 2302d2e..7663678 100644
--- a/.venv/Lib/site-packages/muxtools/muxing/tmdb.py
+++ b/.venv/Lib/site-packages/muxtools/muxing/tmdb.py
@@ -158,8 +158,7 @@ class TmdbConfig:
             value = ET.SubElement(simple, "String")
             value.text = str(v)

-        with open(outfile, "w") as f:
-            ET.ElementTree(main).write(f, encoding="unicode")
+        ET.ElementTree(main).write(outfile, encoding="utf-8")

         return outfile
