diff --git a/muxtools/subtitle/font.py b/muxtools/subtitle/font.py
index 07214e3..4ecc34e 100644
--- a/muxtools/subtitle/font.py
+++ b/muxtools/subtitle/font.py
@@ -1,7 +1,7 @@
-import os
 import shutil
 import logging
 from pathlib import Path
+from pathvalidate import sanitize_filename

 from .sub import SubFile, FontFile
 from ..utils.env import get_workdir
@@ -67,7 +67,7 @@ def collect_fonts(sub: SubFile, use_system_fonts: bool = True, additional_fonts:

             debug(f"Found font '{fontname}'.", collect_fonts)
             fontpath = Path(query.font.filename)
-            outpath = os.path.join(get_workdir(), f"{fontname}{fontpath.suffix}")
+            outpath = get_workdir().joinpath(f"{sanitize_filename(fontname)}{fontpath.suffix}")
             if not Path(outpath).exists():
                 shutil.copy(fontpath, outpath)

