diff --git a/.venv/Lib/site-packages/vsdeband/noise.py b/.venv/Lib/site-packages/vsdeband/noise.py
index 70969bf..6f17d67 100644
--- a/.venv/Lib/site-packages/vsdeband/noise.py
+++ b/.venv/Lib/site-packages/vsdeband/noise.py
@@ -76,7 +76,7 @@ class GrainPP(_gpp):


 FadeLimits = tuple[int | Iterable[int] | None, int | Iterable[int] | None]
-GrainPostProcessT = ResolverOneClipArgs | ResolverTwoClipsArgs | str | GrainPP | GrainPP.Resolver
+GrainPostProcessT = type[ResolverOneClipArgs | ResolverTwoClipsArgs | str | GrainPP | GrainPP.Resolver]
 GrainPostProcessesT = GrainPostProcessT | list[GrainPostProcessT]


